app/app.go

```diff
@@ -89,37 +88,16 @@ func (app *App) configureStorage() {
 	}
 
 	app.configureBucket()
-	if app.Defaults.CassandraEnabled {
-		app.configureCassandra()
-	}
 }
 
 func (app *App) configureDefaults() {
 	app.Defaults = &models.Defaults{
-		BucketQuantityOnSelect:  app.Config.GetInt("cassandra.bucket.quantity"),
-		LimitOfMessages:         app.Config.GetInt64("cassandra.messages.limit"),
+		LimitOfMessages:         app.Config.GetInt64("mongo.messages.limit"),
 		MongoEnabled:            app.Config.GetBool("mongo.messages.enabled"),
```

Comment:
Since we have only Mongo now, this configuration has lost its purpose. You can remove it.

Reply:
Absolutely! 💃 Just updated it!